XamarinCommunityToolkit/Behaviors/TagBehavior.shared.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Behaviors
+{
+    public class TagBehavior : BaseBehavior<Label>
+    {
+        public IList<TagType> TagTypes { get; } = new List<TagType>();
+
+        string RegexPattern => $@"[{string.Join(string.Empty, TagTypes.Select(s => s.Symbol))}]\w+";
+
+        public static readonly BindableProperty CommandProperty =
+         BindableProperty.Create(nameof(Command), typeof(ICommand), typeof(TagBehavior), null, defaultBindingMode: BindingMode.TwoWay);
+
+        public ICommand Command
+        {
+            get => (ICommand)GetValue(CommandProperty);
+            set => SetValue(CommandProperty, value);
+        }
+
+        protected override void OnAttachedTo(Label bindable)
+        {
+            OnViewPropertyChanged(bindable, new PropertyChangedEventArgs(nameof(Label.FormattedText)));
```

Comment:
Would be better to extract login into the separate private method and call that method here.

Reply:
Fixed